src/js/bindings/role.spec.js

```diff
@@ -22,7 +22,7 @@ const characters = [
 ];
 
 describe('role.js', () => {
-  describe('getSelectedFilterRole()', () => {
+  describe.skip('getSelectedFilterRole()', () => {
```

Comment:
Is this something still in progress or is this meant to (theoretically, since I know y'all are moving on to the capstone) be addressed in a future card?

Reply:
Yes this is going to be addressed in the future. Jordan told me to skip this test for now, so that the checks pass. Bennett might be rewriting most of this file because we totally changed what his initial `role.js` file was doing. These checks were redundant compared to the `characterFilter` tests, so we'll have to change them to better fit what `role.js` is doing now.